### PR TITLE
Fixes #5 Adding new model for project data

### LINKED ISSUE #5: Add a model for project data
We can use the Open Humans API's [project information endpoint](https://www.openhumans.org/direct-sharing/on-site-data-access/#api-project-info) to identify what project ID a master token refers to.

Create a `Project` model in this Django app with fields corresponding to this information. Treat the project's ID returned by the API as a unique identifier of the project.

Modify LoginView to immediately use the master token to query the API, and then either (1) create/save a new Project in the database if one doesn't already exist, or (2) update information for a Project in the database, if it already exists.

### PR DESCRIPTION
**Pull request by eeshadutta**
WIP Adding Model for Project Data

I have added a new model for the project data. 
project_db
project_db2

Whenever the user enters the token, an API call is being made and the data is either stored or updated in the database.

In addition, I have also restructured the project to follow Django 2.0 guidelines.
The project name is oh-proj-management and the app is named project.
The project structure is as follows-
filestructure

### CODE REVIEW

**Comment by jasmine1997** [comment]
Hi, you should probably add `.idea/` to `git/info/exclude` or `.gitignore` to avoid committing your IDE configuration.

**Comment by fiennyangeln** [comment]
Hi, I dont think we should remove `env.example` and put the secretkey, debug and other value inside the `oh_proj_management/settings.py` file. As far as I read, using environment variable might be better in case later we are on production and we would not want our key to be exposed. It also helps to separate production and development whereby you might want to set DEBUG value differently.

**Comment by rosygupta** [comment]
Hi @eeshadutta,

Apart from the functionalities, some suggestions for solving the issue:
1. You MIGHT not want to push the addition of .idea in the .gitignore file. You could keep it on your local system since the .idea folder is specific to some IDEs only. 
2. Also, you could update your forked repo so that it consists of the latest changes as per the branch you are creating a pull request against. You are currently getting conflicts in some files. After updating your forked repo you would probably not face these problems.
3. You could use `git diff` for each file before staging your changes for commit (if not doing already). This will help you see the files being committed. Any file modified/deleted unnecessarily should not be committed. You should try to change the least amount of code and fix the issue at the same time. 

Please correct me if I'm wrong anywhere. But hoping this helps :)

File: README.md [hunk 1]
```diff
@@ -33,13 +33,11 @@ Read [this issue string](https://github.com/OpenHumans/open-humans/issues/690) f
...
-  This project uses pipenv — the officially recommended Python packaging tool from Python.org. Read more about pipenv here: https://docs.pipenv.org/
+**Virtual environments:** If you're new to developing Python projects, we strongly recommend you learn about using virtual environments for your work. Either virtualenv (with virtualenvwrapper recommended) or conda are reasonable options.
+ * About virtualenv and virtualenvwrapper: http://docs.python-guide.org/en/latest/dev/virtualenvs/
+ * About Conda's virtual environments: https://conda.io/docs/user-guide/tasks/manage-environments.html
 
-  * Install pipenv with `pip install pipenv`
-  * Activate the virtual environment with `pipenv shell`
-
-  When developing on your own machine, you should be installing requirements from `Pipfile` within a virtual environment – not globally.
+When developing on your own machine, you should be installing requirements from `requirements.txt` within a virtual environment – not globally.
```
Comment (mcescalante) [inline README.md]: We recently switched from `virtualenv` to `pipenv` in this PR: https://github.com/OpenHumans/oh-proj-management/pull/9 so we would like to keep the pipenv instructions

**Pull request closed without merging**